src/main/java/io/vertx/core/file/impl/FileSystemImpl.java

```diff
@@ -72,32 +74,42 @@ public FileSystemImpl(VertxInternal vertx) {
   }
 
   public FileSystem copy(String from, String to, Handler<AsyncResult<Void>> handler) {
-    copyInternal(from, to, handler).run();
+    return copy(from, to, CopyOptions.DEFAULT_OPTIONS, handler);
+  }
+
+  @Override
+  public FileSystem copy(String from, String to, CopyOptions options, Handler<AsyncResult<Void>> handler) {
+    copyInternal(from, to, options, handler).run();
     return this;
   }
 
   public FileSystem copyBlocking(String from, String to) {
-    copyInternal(from, to, null).perform();
+    copyInternal(from, to, CopyOptions.DEFAULT_OPTIONS, null).perform();
     return this;
   }
 
   public FileSystem copyRecursive(String from, String to, boolean recursive, Handler<AsyncResult<Void>> handler) {
-    copyInternal(from, to, recursive, handler).run();
+    copyRecursiveInternal(from, to, recursive, handler).run();
```

Comment:
Because the two methods do completely different things. Separating them makes the code easier to read.